sanitiser/_geo_search.js

```diff
@@ -9,10 +11,22 @@ module.exports = function sanitize( raw, clean ){
   // error & warning messages
   var messages = { errors: [], warnings: [] };
 
+  // disallow specifying both focus.point and focus.viewport
+  if ( ( raw['focus.viewport.min_lat'] ||
```

Comment:
these should be key checks using `hasOwnProperty()` as value checks will fail for zero values such as `0,0`